src/main/java/edu/harvard/iq/dataverse/util/json/JsonParser.java

```diff
@@ -1,23 +1,7 @@
 package edu.harvard.iq.dataverse.util.json;
 
 import com.google.gson.Gson;
-import edu.harvard.iq.dataverse.ControlledVocabularyValue;
```

Comment:
@mderuijter is your IDE set to replace imports with `*` imports? If so, please turn that off.

Reply:
@janvanmansum I think so... Let me check